README.md

```diff
@@ -93,8 +93,7 @@ should be encrypted), use method `.create_user()` for this purpose. Also,
 set fields if appropriate optional values are provided.
    - `get_user`, takes `user_id`, returns user with the given id.
    - `update_user`, takes `user_id`, optional parameters `username`, `password`, 
-`email`, 
-`first_name`, `last_name`. Update user with
+`email`, `first_name`, `last_name`. Update user with
```

Comment:
Remember not to touch readme. I won't reject the task because of this, but remember for future)